jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -1040,7 +1040,8 @@ bool java_bytecode_languaget::convert_single_method(
       throw_assertion_error,
       std::move(needed_lazy_methods),
       string_preprocess,
-      class_hierarchy);
+      class_hierarchy,
+      threading_support);
```

Comment:
I can't work out how this is ever set? Did this flag always exist and just wasn't used?

Reply:
It existed before (and was/is used for other stuff), it is set parse_options if the java-threading flag is specified.